assets/search/components/SearchResultsBar/SearchResultsFiltersRow.tsx

```diff
@@ -56,6 +57,7 @@ export function SearchResultsFiltersRow({readonly, searchParams, filterGroups, t
                                 ...searchParams.created,
                                 from: null,
                             });
+                            refresh?.();
```

Comment:
can't`setCreatedFilter` handle the refreshing?

Reply:
`setCreatedFilter` currently doesn't. Even if I made it do the refreshing it's not used in all the places so it's not suitable.